### PR TITLE
completed javascript-data-models exercise.

### PR DESCRIPTION
**Pull request by vkahlon**
p1

p2
p3
>
p4

### CODE REVIEW

**Comment by vkahlon** [comment]
which means no more easter eggs.

File: javascript-data-model/order-history.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    multipleItems: false,
+    title: 'Javascript for Impatient programmers',
+    isAuthor: true,
+    author: 'Rauschmayer, Dr. Axel',
+    returnClosed: true,
+    windowDate: 'Sep 7, 2020',
+    price: 31.55,
+    writtenReview: false,
+    img: 'bearNoob'
+  },
```
Comment (codingcodymiller) [inline javascript-data-model/order-history.js:18]: In this case, the data for the order itself is intermixed with the data concerning the items ordered, which makes this data model ambiguous and difficult to use. You should have a property of the object for the order that contains the data for the item(s) ordered by the user.

File: javascript-data-model/order-history.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    img: 'gcAdapter',
+
+    title2: 'The Art if SQL',
+    isAuthor2: true,
+    author2: 'Faroult, Stephane',
+    returnClosed2: true,
+    windowDate2: 'August 4, 2020',
+    price2: 33.99,
+    writtenReview2: false,
+    img2: 'sqlBook'
```
Comment (codingcodymiller) [inline javascript-data-model/order-history.js:78]: Hooooo boy, properties with a `2` at the end? That doesn't sound easily maintainable, you should organize this in a way where each item can have all of its data enclosed together without worrying about intermixing the data with data for other items.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-data-model/order-history.js (hunk 2 above)
Comment (vkahlon) [reply javascript-data-model/order-history.js]: Resubbed and essentially overhauled the entire assignment.

**Review by codingcodymiller — APPROVED** [review]
Great work!

**Pull request merged**